pkg/version/check.go

```diff
@@ -86,22 +85,35 @@ func CheckClusterVersion(
 		log.Warn("check TiKV version failed", zap.Error(err))
 	}
 
-	httpCli, err := httputil.NewClient(credential)
-	if err != nil {
-		return err
+	for _, pdAddr := range pdAddrs {
+		err = CheckPDVersion(ctx, pdAddr, credential)
+		if err == nil {
+			break
+		}
 	}
+
+	return err
+}
+
+// CheckPDVersion check PD version.
+func CheckPDVersion(ctx context.Context, pdAddr string, credential *security.Credential) error {
 	// See more: https://github.com/pingcap/pd/blob/v4.0.0-rc.1/server/api/version.go
 	pdVer := struct {
 		Version string `json:"version"`
 	}{}
 
+	httpClient, err := httputil.NewClient(credential)
```

Comment:
I would prefer to name it `cli`, we know that it's an HTTP client by the right-hand side.

Reply:
cli is a very confusing acronym (first thought is command line interface), and it is not a recognized acronym for the client, it does not have a consensus acronym. So I prefer to change it to client to reduce confusion.